src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -926,6 +926,25 @@ class ApplicationManagerImpl
    */
   void RemoveAppFromTTSGlobalPropertiesList(const uint32_t app_id) OVERRIDE;
 
+  /**
+   * @brief Resets application's global properties to default values
+   * returning struct that indicates which properties have been
+   * successfully reset.
+   * @param container with global properties to reset
+   * @return struct with flags indicating global properties reset
+   */
+  virtual ResetGlobalPropertiesResult ResetGlobalProperties(
+      const smart_objects::SmartObject& global_properties_ids,
```

Comment:
@mked-luxoft why global properties ids? is it list of integers?

Reply:
@LuxoftAKutsan it is an array smart object with list of enum constants representing global properties IDs